### PR TITLE
`indexer-cli`, `indexer-common`: Introduce the `--first` CLI option

### PR DESCRIPTION
**Pull request by tilacog**
This PRs allows the `Action` resolver to limit the number of records returned by the query.

### CODE REVIEW

**Comment by tilacog** [comment]
The `actions get` command fails with this message for all possible invocations involving `--first`
```
✖ [GraphQL] Variable "$first" expected value of type "Integer" which cannot be used as an input type.
```

**Comment by tilacog** [comment]
Working as expected now

File: packages/indexer-cli/src/commands/indexer/actions/get.ts [hunk 1]
```diff
@@ -21,11 +21,12 @@ ${chalk.dim('Options:')}
 
   -h, --help                                                        Show usage information
       --type    allocate|unallocate|reallocate|collect              Filter by type
-      --status  queued|approved|pending|success|failed|canceled     Filter by status 
+      --status  queued|approved|pending|success|failed|canceled     Filter by status
       --source <source>                                             Fetch only actions queued by a specific source
       --reason <reason>                                             Fetch only actions queued for a specific reason
       --orderBy id|deploymentID|amount|priority|...|updatedAt       Order actions by a specific field (default: id)
       --orderDirection asc|desc                                     Order direction (default: desc)
+      --first [N]                                                   Fetch only the N first records (defaults to 1, returns all records if omitted)
```
Comment (fordN) [inline packages/indexer-cli/src/commands/indexer/actions/get.ts:29]: What do you think about defaulting to 100? That way we don't cause significant changes in the current default behavior.

File: packages/indexer-cli/src/commands/indexer/actions/get.ts [hunk 2]
```diff
@@ -91,6 +103,22 @@ module.exports = {
           ? OrderDirection[orderDirection.toUpperCase() as keyof typeof OrderDirection]
           : OrderDirection.DESC
       }
+
+      switch (typeof first) {
+        case 'boolean':
+          // can only be `true` at this point
+          fetchFirst = 1
```
Comment (fordN) [inline packages/indexer-cli/src/commands/indexer/actions/get.ts:110]: Similar comment to above. I'd suggest defaulting to `100` to maintain a smooth transition from previous behavior.

**Review by fordN — APPROVED** [review]
I'd suggest updating the default behavior. Otherwise looks good!

**Comment by tilacog** [comment]
I'll rebase to tidy up commit history.

**Review by fordN — APPROVED** [review]
👍

**Pull request merged**